src/api.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+const getArticles = () => {
+  return axios
```

Comment:
Nice extraction of API logic to this file, can take it a step further using axios create if you want (check out the docs, will allow you to configure baseURL to make code more dry for example)